src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -384,7 +390,7 @@ private EmailWrapper generateFeedbackSessionEmailBaseForInstructorReminders(
                 "${submitUrl}", submitUrl,
                 "${reportUrl}", reportUrl,
                 "${feedbackAction}", FEEDBACK_ACTION_SUBMIT_EDIT_OR_VIEW,
-                "${additionalContactInformation}", additionalContactInformation);
+                "${additionalContactInformation}", instructorJoinReminderFragment + additionalContactInformation);
```

Comment:
I dont think combining `instructorJoinReminderFragment` with `additionalContactInformation` is a good idea, lets keep them separate

Reply:
I've made the requested changes, there's an extra placeholder {additionalNotes} before {additionalContactInformation}